### DISCUSSION TITLE
Seeking Guidance on Frame Synchronization Implementation in the Project

### DISCUSSION THREAD

**Discussion by realchenhao** [srsran/srsRAN_Project#507] [General Help]
Dear Community Members,
I hope you're doing well. As a newcomer to this project, I'm looking for guidance on implementing frame synchronization. I have a couple of questions:

1. How is frame synchronization managed between base stations in this project? Does it use relative synchronization or absolute synchronization? Relative synchronization involves aligning frames based on a reference point or a common clock, while absolute synchronization aligns frames precisely according to a global clock or timing signal. I'd appreciate any insights on how these concepts are applied here.
2. Regarding the part of the code that handles frame synchronization configurations, I am interested in knowing where it is implemented. Could someone please direct me and provide insights on how I can effectively modify this section?

Any assistance or advice you can offer would be greatly appreciated. Thanks for taking the time to help a beginner like me navigate through this project.

**Comment by pgawlowicz** [comment]
@realchenhao any update on this issue?
